### DISCUSSION TITLE
Any plans to support focus outlines?

### DISCUSSION THREAD

**Discussion by Gerbert-Kaandorp** [beercss/beercss#371] [General]
Hello,
I love the minimalistic approach of this framework, very nice!
I am testing it out and my biggest issue is the focus outlines. 
It doesn't seem to look like the specs.

what I am missing is something like this:
      button:focus {
        outline: solid var(--primary);
        outline-offset: 0.2rem;
      }
On all relevant components :)

Or is there a reason it looks the way it does currently?

Thanks for reading.  
Cheers!

**Comment by Gerbert-Kaandorp** [comment]
![image](https://github.com/user-attachments/assets/e7da50cc-2e2a-47b1-97aa-33bf869aa508)

**Comment by leonardorafael** [comment]
Seems nice and follows the Material Design 3 too. Thanks for feedback 🍺

**Comment by leonardorafael** [comment]
Done in https://github.com/beercss/beercss/releases/tag/v3.7.1 (on buttons and links only)

**Comment by Gerbert-Kaandorp** [comment]
omg that was fast 

Thanks you!
 🙇
